src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -1190,15 +1190,14 @@ void M()
     Predecessors (1)
         [0]
     Statements (1)
-        IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'bool condition = false;')
-          IVariableDeclarationOperation (1 declarators) (OperationKind.VariableDeclaration, Type: null) (Syntax: 'bool condition = false')
-            Declarators:
-                IVariableDeclaratorOperation (Symbol: System.Boolean condition) (OperationKind.VariableDeclarator, Type: null) (Syntax: 'condition = false')
-                  Initializer: 
-                    IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null) (Syntax: '= false')
-                      ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: False) (Syntax: 'false')
-            Initializer: 
-              null
+        IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: 'condition = false')
+          Expression: 
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Boolean, IsImplicit) (Syntax: 'condition = false')
+              Left: 
+                ILocalReferenceOperation: condition (IsDeclaration: True) (OperationKind.LocalReference, Type: System.Boolean, IsImplicit) (Syntax: 'condition = false')
+              Right: 
+                IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null) (Syntax: '= false')
```

Comment:
Why are we retaining the `IVariableInitializerOperation`, which is normally expected to be parented by a variable declaration? I thought we were going to do the reverse lowering - remove `IVariableInitializerOperation` nodes and add subsequent simple assignments operations for initialization.

Reply:
Note that not having to deal with different kind of initializers (simple variable/param/member initializer, array initializer, object initializer, collection initializer, member initializer, etc) does simplify flow analysis significantly, and lowering them into simple assignments followed by a declaration is definitely a value add.

Reply:
My plan this PR was to only handle `IVariableDeclarationGroup`, `IVariableDeclaration`, and `IVariableDeclarator`. If reviewing it is easier, I can also add in `IVariableInitializerOperation`, but I figured that would be handled in the same PR as the other initializer operations.

Reply:
If lowering `IVariableInitializerOperation` is coming in next PR, that is fine. However, I am still not convinced why get rid of the declaration operations. What is the motivating factor behind this change?